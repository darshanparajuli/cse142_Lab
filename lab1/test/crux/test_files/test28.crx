123.445aaa
123bbb
