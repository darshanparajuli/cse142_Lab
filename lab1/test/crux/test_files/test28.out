FLOAT(123.445)(lineNum:1, charPos:1)
IDENTIFIER(aaa)(lineNum:1, charPos:8)
INTEGER(123)(lineNum:2, charPos:1)
IDENTIFIER(bbb)(lineNum:2, charPos:4)
EOF(lineNum:3, charPos:1)
